core/lib/spree/testing_support/factories/address_factory.rb

```diff
@@ -11,7 +11,7 @@
     end
 
     firstname 'John'
-    lastname 'Doe'
+    lastname nil
```

Comment:
I'm not sure what I think about the theory of factories being intended to make typical vs minimal objects -- perhaps there sometimes need to be mutiple versions of a factory to handle multiple cases (see product, with base_product, product, and then various even more complicated projects), but I have no strong feelings either way about this particular change, it seems fine. Someone can later add a `person_with_lots_of_stuff` factory if needed.